scripts/g_r.py

```diff
@@ -0,0 +1,60 @@
+import datetime
+import pandas
+from collections import Counter
+from sqlalchemy import create_engine
+engine = create_engine('mysql://root@localhost:3306/pantherbot_test', echo=False)
```

Comment:
Engine is being connected in two places? seems like a bad idea, also this is again for local only and on your test db, will fail in production